plugin/helpers.go

```diff
@@ -12,7 +12,7 @@ type Helpers interface {
 	EnsureBot(bot *model.Bot) (string, error)
 
 	// KVGetJSON retrievs a value based on the key.
-	KVGetJSON(key string, value interface{}) error
+	KVGetJSON(key string, value interface{}) (bool, error)
```

Comment:
Kinda obvious when reading through the code what the `bool` return value means, but i wonder if the doc on line 14 should be updated to describe the new behaviour?

Reply:
Also, we could consider improving the docstring to remind people that the retrieved value will be 'returned' via the `value` param? Basic Go stuff, but could help community members?